tests/backup/backup_test.go

```diff
@@ -2722,6 +2722,260 @@ var _ = Describe("{BackupAlternatingBetweenLockedAndUnlockedBuckets}", func() {
+					expected := api.BackupInfo_StatusInfo_Success
+					if actual != expected {
+						return "", true, fmt.Errorf("backup status for [%s] expected was [%s] but got [%s]", recentBackupName, expected, actual)
+					}
+					return "", false, nil
+				}
+				_, err = task.DoRetryWithTimeout(backupSuccessCheck, 10*time.Minute, 30*time.Second)
+				log.FailOnError(err, "Backup with name %s was not successful", recentBackupName)
+				log.InfoD("New backup - %s is successful from schedule backup ", recentBackupName)
+			}
+			log.InfoD("All Accesses are toggled and operations are performed")
+
+		})
+
+	})
+	JustAfterEach(func() {
+		defer EndTorpedoTest()
+		log.InfoD("End test")
+		ctx, err := backup.GetAdminCtxFromSecret()
+		log.FailOnError(err, "Fetching px-central-admin ctx")
+
+		//Get scheduleBackupUid
+		log.InfoD("Get scheduleBackupUid")
+		scheduleBackupyUid, err := GetbackupScheduleUID(backupName, orgID, ctx)
+		log.InfoD("scheduleBackupyUid - %v", scheduleBackupyUid)
+		log.FailOnError(err, "Error deleting Schedule backup %v", backupName)
+
+		//Delete Schedule Backup-
+		log.InfoD("Delete Schedule Backup-")
+		err = DeleteScheduleBackup(backupName, scheduleBackupyUid, periodicPolicyName, schPolicyUid, orgID)
+		log.FailOnError(err, "Error deleting Schedule backup %v", backupName)
+
+		//GetAll backups -
+		backupNames, err := GetAllBackupsAdmin()
+		log.FailOnError(err, "Fetching admin backups")
+
+		//Delete Backup
+		backupDriver := Inst().Backup
+		for _, backupName := range backupNames {
+			log.InfoD("Deleting backup - %v", backupName)
+			backupUID, err := backupDriver.GetBackupUID(ctx, backupName, orgID)
+			log.FailOnError(err, "Failed while trying to get backup UID for - %s", backupName)
+			log.Infof("About to delete backup - %s", backupName)
+			backupDeleteResponse, err := DeleteBackup(backupName, backupUID, orgID, ctx)
+			log.FailOnError(err, "Backup [%s] could not be deleted with delete response %s", backupName, backupDeleteResponse)
+		}
+
+		log.InfoD("Deleting restore for user")
+		ctxNonAdmin, err := backup.GetNonAdminCtx(userName, password)
+		log.FailOnError(err, "Fetching %s ctx", userName)
+
+		for _, restore := range restoreNames {
+			err := DeleteRestore(restore, orgID, ctxNonAdmin)
+			log.FailOnError(err, "Deleting User Restore")
+		}
+
+		//Deleting user
+		err = backup.DeleteUser(userName)
+		log.FailOnError(err, "Error deleting user %v", userName)
+
+		log.InfoD("Deleting the deployed apps after the testcase")
+		for i := 0; i < len(contexts); i++ {
+			opts := make(map[string]bool)
+			opts[SkipClusterScopedObjects] = true
+			taskName := fmt.Sprintf("%s-%d", taskNamePrefix, i)
+			err := Inst().S.Destroy(contexts[i], opts)
+			dash.VerifySafely(err, nil, fmt.Sprintf("Verify destroying app %s, Err: %v", taskName, err))
+		}
+
+		log.Infof("Deleting registered clusters for admin context")
+		DeleteCluster(SourceClusterName, orgID, ctx)
+		DeleteCluster(destinationClusterName, orgID, ctx)
```

Comment:
These steps are not required as `DeleteCloudAccounts` will delete both the source and destination cluster